doc/en/components/grids/_shared/excel-style-filtering.md

```diff
@@ -0,0 +1,1017 @@
+            <div class="cell__inner_2">
+                <img [src]="cell.value" class="photo" />
+            </div>
+        </ng-template>
+    </igx-column>
+    <igx-column field="Debut" [filterable]='true' [disablePinning]="true" [disableHiding]="true"></igx-column>
+    <igx-column field="Grammy Nominations" [filterable]='true' [dataType]="'number'" [sortable]="false"></igx-column>
+    <igx-column field="Grammy Awards" [filterable]='true' [dataType]="'number'"></igx-column>
+
+    <igx-row-island [key]="'Albums'" [autoGenerate]="false" [allowFiltering]='true' filterMode="excelStyleFilter">
+        <igx-column field="Album" [filterable]='true'></igx-column>
+        <igx-column field="Launch Date" [filterable]='true' [dataType]="'date'"></igx-column>
+        <igx-column field="Billboard Review" [filterable]='true' [dataType]="'number'"></igx-column>
+        <igx-column field="US Billboard 200" [filterable]='true' [dataType]="'number'"></igx-column>
+    <igx-row-island [key]="'Songs'" [autoGenerate]="false" >
+            <igx-column field="No."></igx-column>
+            <igx-column field="Title"></igx-column>
+            <igx-column field="Released"></igx-column>
+            <igx-column field="Genre"></igx-column>
+    </igx-row-island>
+    </igx-row-island>
+
+    <igx-row-island [key]="'Tours'" [autoGenerate]="false">
+        <igx-column field="Tour"></igx-column>
+        <igx-column field="Started on"></igx-column>
+        <igx-column field="Location"></igx-column>
+        <igx-column field="Headliner"></igx-column>
+    </igx-row-island>
+</igx-hierarchical-grid>
+```
+<!-- end: Angular -->
+
+
+```razor
+Add blazor snippets here
+```
+
+In the sample below 'Artist' column have all three features enabled, 'Debut' have all three disabled, 'Grammy Nominations' has only pinning and hiding.
+
+<!-- ComponentEnd: HierarchicalGrid -->
+
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height:620px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-excel-style-filtering-sample-2"
+           github-src="{ComponentSample}/excel-style-filtering-sample-2" >
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-excel-style-filtering-sample-2"
+           github-src="{ComponentSample}/excel-style-filtering-sample-2" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-excel-style-filtering-sample-2"
+           github-src="{ComponentSample}/excel-style-filtering-sample-2" >
+
+</code-view>
+
+<!-- ComponentEnd: HierarchicalGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.